routing/routes.go

```diff
@@ -1420,6 +1420,24 @@ func NewRoutes() (r *Routes) {
 			handlers.OutboundTrafficPolicyMode,
 			true,
 		},
+		// GET /api/mesh/resources/thresholds
```

Comment:
Should this be ` swagger:route GET /api/mesh/resources/thresholds`  ?

Reply:
Yes, I forgot to add it, although we are not using Swagger anymore,  I added in the same fashion as others just in case we will use it again.